cli/jobs/train/r/accident-prediction/accident_r_job.yml

```diff
@@ -0,0 +1,18 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandJob.schema.json
+experiment_name: r-accidents-example
+code:
+  local_path: ./src
+command: >-
+  Rscript accidents.R
+  --data {inputs.training_data}
+environment:
+  docker:
+    build:
+      dockerfile: file:Dockerfile
+compute:
+  target: azureml:goazurego
```

Comment:
not a compute target in the workspace, see `setup.sh` (you probably want `cpu-cluster`)